docs/eventing/broker-trigger.md

```diff
@@ -0,0 +1,268 @@
+[default provisioner](https://www.knative.dev/docs/eventing/channels/default-channels/)
+for the namespace you are interested in. For development, the
+[`in-memory` `ClusterChannelProvisioner`](https://github.com/knative/eventing/tree/master/config/provisioners/in-memory-channel#deployment-steps)
+is normally used.
+
+#### Changing
+
+**Note** changing the `ClusterChannelProvisioner` of a running `Broker` will
+lose all in-flight events.
+
+If you want to change which `ClusterChannelProvisioner` is used by a given
+`Broker`, then determine if the `spec.channelTemplate` is specified or not.
+
+If `spec.channelTemplate` is specified:
+
+1. Delete the `Broker`.
+1. Create the `Broker` with the updated `spec.channelTemplate`.
+
+If `spec.channelTemplate` is not specified:
+
+1. Change the
+   [default provisioner](https://github.com/knative/docs/blob/master/docs/eventing/channels/default-channels.md#setting-the-default-channel-configuration)
+   for the namespace that `Broker` is in.
+1. Delete and recreate the `Broker`.
+
+### Broker
+
+There are two ways to create a Broker, via [namespace annotation](#annotation)
+or [manual setup](#manual-setup).
+
+Normally the [namespace annotation](#annotation) is used to do this setup.
+
+#### Annotation
+
+The easiest way to get started, is to annotate your namespace (replace `default`
+with the desired namespace):
+
+```shell
+kubectl label namespace default knative-eventing-injection=enabled
+```
+
+This should automatically create the `default` `Broker` in that namespace.
+
+```shell
+kubectl -n default get broker default
+```
+
+#### Manual Setup
+
+In order to setup a `Broker` manually, we must first create the required
+`ServiceAccount` and give it the proper RBAC permissions. This setup is required
+once per namespace. These instructions will use the `default` namespace, but you
+can replace it with any namespace you want to install a `Broker` into.
+
+Create the `ServiceAccount`.
+
+```shell
+kubectl -n default create serviceaccount eventing-broker-filter
+```
+
+Then give it the needed RBAC permissions:
+
+```shell
+kubectl -n default create rolebinding eventing-broker-filter \
+  --clusterrole=eventing-broker-filter \
+  --user=eventing-broker-filter
+```
+
+Note that the previous commands uses three different objects, all named
+`eventing-broker-filter`. The `ClusterRole` is installed with Knative Eventing
+[here](../../config/200-broker-clusterrole.yaml). The `ServiceAccount` was
+created two commands prior. The `RoleBinding` is created with this command.
+
+Now we can create the `Broker`. Note that this example uses the name `default`,
+but could be replaced by any other valid name.
+
+```shell
+cat << EOF | kubectl apply -f -
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Broker
+metadata:
+  namespace: default
+  name: default
+EOF
+```
+
+### Subscriber
+
+Now create some function that wants to receive those events. This document will
```

Comment:
Now create _a_ function .....This document will assume the following manifest describing a Knative service is created but it could be anything...